xl/player/player.py

```diff
@@ -397,7 +397,8 @@ def is_playing(self):
             :returns: whether the player is currently playing
             :rtype: bool
         """
-        return self._engine.get_state() == 'playing'
+        return self._engine.get_state() is not None \
```

Comment:
> `'NoneType' object has no attribute 'get_state'`

Oh, that means it's `self._engine` that is None at that point. You can check that instead.